webrtc.html

```diff
@@ -3179,6 +3160,32 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
 	  </dd>
         </dl>
 
+        <dl class="idl" title="enum RTCRtpDirectionality">
+          <dt>sendrecv</dt>
+          <dd>The <a><code>RTCRtpTransceiver</code></a>'s <a><code>RTCRtpSender</code></a>
+            will offer to send RTP, and will send RTP (<var>active</var> set to "true") if the remote peer accepts.
+            The <a><code>RTCRtpTransceiver</code></a>'s <a><code>RTCRtpReceiver</code></a> will offer to receive RTP,
+            and will receive RTP (<var>active</var> set to "true") if the remote peer accepts.</dd>
+
+          <dt>sendonly</dt>
+          <dd>The <a><code>RTCRtpTransceiver</code></a>'s <a><code>RTCRtpSender</code></a>
+            will offer to send RTP, and will send RTP (<var>active</var> set to "true") if the remote peer accepts.
+            The <a><code>RTCRtpTransceiver</code></a>'s <a><code>RTCRtpReceiver</code></a> will not offer to receive RTP,
+            and will not receive RTP (<var>active</var> set to "false").</dd>
+
+          <dt>recvonly</dt>
+          <dd>The <a><code>RTCRtpTransceiver</code></a>'s <a><code>RTCRtpSender</code></a> will not offer to send RTP,
+            and will not send RTP (<var>active</var> set to "false"). The <a><code>RTCRtpTransceiver</code></a>'s
+            <a><code>RTCRtpReceiver</code></a> will offer to receive RTP, and will receive RTP (<var>active</var>
+            set to "true") if the remote peer accepts.</dd>
+
+          <dt>inactive</dt>
+          <dd>The <a><code>RTCRtpTransceiver</code></a>'s <a><code>RTCRtpSender</code></a> will not offer to send RTP,
+            and will not send RTP (<var>active</var>  set to "false"). The <a><code>RTCRtpTransceiver</code></a>'s 
+            <a><code>RTCRtpReceiver</code></a> will not offer to receive RTP, and will not receive RTP
+            (<var>active</var> set to "false").</dd>
+        </dl>
+
```

Comment:
Repeat: the "Simple Peer-to-peer Example with media before signaling" example needs to be updated.